src/firefly/js/visualize/ui/CatalogTableListField.jsx

```diff
@@ -168,21 +176,31 @@ function handleOnClick(ev, params, fireValueChange) {
 }
 
 // get element data and wrapit around html markup knowing the columns name
-function htmlElementMarkup(element, cols) {
-    let m = '';
+function itemMarkupTransform(element, cols) {
     const url = '<a target=\'_new\' href=\'http://irsa.ipac.caltech.edu/cgi-bin/Gator/nph-dd?mode=xml&catalog=@\'>go to dd</a>';
-    cols.forEach((e, idx) => {
-            if (idx < 10) {
-                //<span class="marked-text" style="font-size:80%;">Rows: </span><span class="normal-text" style="font-size:80%;">18,240</span>
-                m += '<b>' + e.name + '</b>: ' + element.cat[idx];
-                if (e.name.includes('cat')) {
-                    m += ' [ ' + url.replace('@', element.cat[idx]) + ' ]';
+
```

Comment:
use string template here to avoid escaping single-quote.  or, replace inner single-quote with double-quote.